resources/sts/4.12/hypershift/sts_hcp_support_permission_policy.json

```diff
@@ -65,36 +57,6 @@
         "arn:aws:s3:::*image-registry*"
```

Comment:
I think managed-velero* should be removed here as velero is not a thing in HyperShift